pkgs/applications/audio/flacon/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, cmake, qt5, libuchardet, pkgconfig, makeWrapper
+, shntool, flac, opusTools, vorbisTools, mp3gain, lame, wavpack, vorbisgain
+}:
+
+stdenv.mkDerivation rec {
+  name = "flacon-${version}";
+  version = "2.0.1";
+  src = fetchFromGitHub {
+    owner = "flacon";
+    repo = "flacon";
+    rev = "v2.0.1";
+    sha256 = "0hip411k3arb96rnd22ifs9shlv0xmy96hhx1jcwdk48kw8aa9rw";
+  };
+
+  buildInputs = [ cmake qt5.qtbase qt5.qttools libuchardet pkgconfig makeWrapper];
+
+  postInstall = ''
+    wrapProgram $out/bin/flacon \
+      --prefix PATH : "${shntool}/bin:${flac}/bin:${opusTools}/bin:\
```

Comment:
You can use `makeBinPath` here: https://github.com/NixOS/nixpkgs/blob/6f3e72eef6332ae988d213c76f77db65e8897e62/lib/strings.nix#L115-L123